src/utils/arc-image.ts

```diff
@@ -0,0 +1,34 @@
+import { Octokit } from '@octokit/rest';
+import { MAIN_BRANCH, REPOS } from '../constants';
+import { getOctokit } from './octokit';
+
+const WINDOWS_IMAGE_REGEX =
+  /electronarc\.azurecr\.io\/win-actions-runner:main-[a-f0-9]{7}@sha256:[a-f0-9]{64}/;
+const LINUX_IMAGE_REGEX =
```

Comment:
@MarshallOfSound this appears to be unused?

Reply:
Ah yeah, ran out of battery on the plane so pushed what I had when it worked. But I want to make this auto-bump the linux runner image + the runner version in the windows Dockerfile too.

Reply:
I'll remove this for now